app/graphql/albums/resolvers.js

```diff
@@ -1,27 +1,20 @@
-const { allAlbums, albumById, photosByAlbumId } = require('../../services/album');
+const { allAlbums, albumById, photosByAlbumId } = require('../../services/album'),
+  { changeUserIdByArtist } = require('../../helpers');
 
 exports.getAlbum = async (_, params) => {
   const album = await albumById(params.id);
-  const data = {
-    ...album,
-    artist: album.userId,
-    name: album.title
-  };
-  return data;
+  return changeUserIdByArtist(album);
 };
 
 exports.getAlbums = async (_, params) => {
   const tempAlbums = await allAlbums(params);
-  const albums = tempAlbums.map(album => ({
-    ...album,
-    artist: album.userId
-  }));
+  const albums = tempAlbums.map(album => changeUserIdByArtist(album));
   return albums;
 };
 
-exports.getPhotos = async root => {
+exports.getPhotos = root => {
   const { id } = root;
-  const photos = await photosByAlbumId(id);
+  const photos = photosByAlbumId(id);
```

Comment:
This is not what i mean, is not necessary to instance a var for the promise, just do: `return photosByAlbumId(id);`